pom.xml

```diff
@@ -157,4 +120,31 @@
       <scope>test</scope>
     </dependency>
   </dependencies>
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>org.testcontainers</groupId>
+        <artifactId>postgresql</artifactId>
+        <version>${dep.testcontainers.version}</version>
+        <exclusions>
+          <exclusion>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+          </exclusion>
+        </exclusions>
+      </dependency>
+      <dependency>
+        <groupId>org.testcontainers</groupId>
+        <artifactId>testcontainers</artifactId>
+        <version>${dep.testcontainers.version}</version>
+        <exclusions>
+          <exclusion>
+            <groupId>junit</groupId>
```

Comment:
Dumb question, why did this need to be excluded? To make it optional which version I'm guessing?